infra/base-images/base-image/Dockerfile

```diff
@@ -29,3 +29,17 @@ ENV WORK=/work
 ENV PATH="$PATH:/out"
 
 RUN mkdir -p $OUT $SRC $WORK && chmod a+rwx $OUT $SRC $WORK
+
+# Steps for ARM cross-compilation
+RUN apt-get install qemu-user wget xz-utils -y && \
+    cd /opt && \
+    wget https://releases.linaro.org/components/toolchain/binaries/7.5-2019.12/aarch64-linux-gnu/gcc-linaro-7.5.0-2019.12-x86_64_aarch64-linux-gnu.tar.xz && \
```

Comment:
nit: move 7.5-2019.12 into an ENV param AARCH64_LIB_VERSION and use in all places, inclu AARCH64_TOOLCHAIN